crates/state/src/buffer/kv/test.rs

```diff
@@ -1,17 +1,17 @@
 use super::KvBufUsed;
 use crate::test_utils::DbString;
 use crate::{
+    buffer::{kv::generic::KvStoreT, BufferedStore},
     env::{ReadManager, WriteManager},
     error::{DatabaseError, DatabaseResult},
-    buffer::{kv::generic::KvStoreT, BufferedStore},
     test_utils::test_cell_env,
 };
 use rkv::StoreOptions;
 
-#[tokio::test(threaded_scheduler)]
+#[tokio::test(flavor = "multi_thread")]
 async fn kvbuf_scratch_and_persistence() -> DatabaseResult<()> {
     let test_env = test_cell_env();
-let arc = test_env.env();
+    let arc = test_env.env();
```

Comment:
why didn't our ci linting code catch this? is this file not being scanned or something?